src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -248,5 +334,13 @@ protected override Task<CompletionDescription> GetDescriptionWorkerAsync(Documen
                 ? completionListType
                 : null;
         }
+
+        private static INamedTypeSymbol? TryGetTypeWithStaticMembers(ITypeSymbol type)
+        {
+            if (type.TypeKind == TypeKind.Struct || type.TypeKind == TypeKind.Class)
+                return type as INamedTypeSymbol;
```

Comment:
does this match records?

Reply:
There is no `TypeKind.Record` so I'm assuming yes?

Reply:
➡️ Test now added